filesystem/fs.go

```diff
@@ -76,11 +76,13 @@ func (cfs CFS) create(parentID uint64, name string, node *Node) error {
 	if err != nil {
 		return err
 	}
-	const sql = `
-INSERT INTO fs.inode VALUES ($1, $2);
-INSERT INTO fs.namespace VALUES ($3, $4, $1);
-`
-	if _, err := tx.Exec(sql, node.ID, inode, parentID, name); err != nil {
+	const insertNode = `INSERT INTO fs.inode VALUES ($1, $2)`
+	if _, err := tx.Exec(insertNode, node.ID, inode); err != nil {
+		_ = tx.Rollback()
+		return err
+	}
+	const insertNamespace = `INSERT INTO fs.namespace VALUES ($1, $2, $3)`
+	if _, err := tx.Exec(insertNamespace, parentID, name, node.ID); err != nil {
```

Comment:
My understanding is that the one-statement limitations is a part of the extended query protocol. While the query to be executed could contain multiple statements, only one set of results can be returned. Perhaps I'm misreading the docs.